tools/features/com.liferay.ide.hidden/feature.xml

```diff
@@ -107,4 +107,32 @@
          version="0.0.0"
          unpack="false"/>
 
+   <plugin
+         id="com.liferay.ide.server.core.tests"
+         download-size="0"
+         install-size="0"
+         version="0.0.0"
+         unpack="false"/>
+
+   <plugin
+         id="com.liferay.ide.project.core.tests"
+         download-size="0"
+         install-size="0"
+         version="0.0.0"
+         unpack="false"/>
+
+   <plugin
```

Comment:
Try adding this to the build: http://oss.opensagres.fr/tern.repository/0.8.0/

That updatesite should have minimal-json in it and it should be resolving automatically.